common/lib/xmodule/xmodule/tests/test_lti_unit.py

```diff
@@ -53,11 +56,17 @@ def setUp(self):
                   </imsx_POXBody>
                 </imsx_POXEnvelopeRequest>
             """)
+        self.system = get_test_system()
         self.system.get_real_user = Mock()
         self.system.publish = Mock()
         self.system.rebind_noauth_module_to_user = Mock()
+        self.user_id = self.system.anonymous_student_id
 
-        self.user_id = self.xmodule.runtime.anonymous_student_id
+        self.xmodule = LTIBlock(
```

Comment:
Nit: Consider swapping this name out for `xblock` or `lti_block` or something. But if it's too time consuming, don't bother.

Reply:
To keep the diffs small and easy to understand while doing the risky conversions I have generally avoided renaming variables. But am planning to do a search and replace in the repo for `module`/`descriptor` when we remove the XModule/Descriptor classes. Does that make sense?

Reply:
Sounds great 👍